src/main/vcp_hal/usbd_cdc_interface.h

```diff
@@ -65,6 +65,18 @@
    The period depends on CDC_POLLING_INTERVAL */
 #define CDC_POLLING_INTERVAL             10 /* in ms. The max is 65 and the min is 1 */
 
+/* Exported typef ------------------------------------------------------------*/
+/* The following structures groups all needed parameters to be configured for the
+   ComPort. These parameters can modified on the fly by the host through CDC class
+   command class requests. */
+typedef struct __attribute__ ((packed))
```

Comment:
This struct is available in HAL, with different name. But it's not a big problem ...

Reply:
USBD_CDC_LineCodingTypeDef isn't packed which is the point here.